### ISSUE TITLE
Pause changes to off on second call

### ISSUE THREAD

**Issue by Hootie81** [vinteo/hass-opensprinkler#276]
Thanks for the pause feature.. I've been waiting patiently

One thing that took me a minute to figure out when i was setting up a google assistant routine, if i make a service call the first time it works as expected, however if i call it again while the first pause is running the pause is ended. it also takes a few seconds to update which confused me more, because pause time remaining went to unknown but the pause state stayed on for quite a few seconds.

is it possible to change this behavior so a second call will just update the pause time to whatever is in the call? and possibly add a resume service? basically the same call with 1 second pause or something

the main thing i want to do is be able to yell at my phone when the water pressure drops when I'm in the shower.. and this update solves that! really appreciate your work on this integration.

**Comment by curlydingo** [comment]
Hmmm, that's interesting. I hadn't tried calling the pause service twice. Will look in to it.

I debated adding a 'resume' service but setting a pause duration of 0 does this anyway, so it seemed overkill.

**Comment by EdLeckert** [comment]
This appears to be a bug in the firmware. Issuing `http://<IP>/pq?dur=600&pw=<pw>` twice in a row disables the pause on the second call.

**Comment by curlydingo** [comment]
Agreed. Looking at the firmware  code though it’s more than a bug; pause there is designed as a toggle rather than a 0-to-reset paradigm that the docs talk about. I decided the above pull request was our best option to fix it quickly.
